lib/flutter_billing.dart

```diff
@@ -143,6 +143,14 @@ class Billing {
     });
   }
 
+  /// Grab in app receipt for iOS.
```

Comment:
I think we should implement this for Android. We could define a class `Receipt` with a value and unsupported. Something like this?

```dart
class Receipt {
  static final Receipt _unsupported = new Receipt._();

  Receipt._({this.payload});

  factory Receipt.success(String payload) {
    return Receipt._(payload: payload);
  }

  factory Receipt.unsupported() {
    return _unsupported;
  }

  final String payload;
}
```

For this type may need to generate `operator ==`, `hashCode`, and `toString`

Reply:
Well the problem is: it isn't a simple call on Android side like mentionned on #6.

We need to change the way a purchase success is returned, with receipt informations, no more checking a list.

Reply:
Would really enjoy some help on this part, since I'm not so familiar with Dart, Objective-C & Java. That's a whole stuff :)